<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionAnswerClass</class>
 <widget class="QMainWindow" name="QuestionAnswerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuestionAnswer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QListView" name="questionView"/>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QListWidget" name="answerList"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="questionEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add Question</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="answerEdit"/>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="answerButton">
      <property name="text">
       <string>Add Answer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QuestionAnswer.qrc"/>
 </resources>
 <connections/>
</ui>
